packages/core/src/http-client/http-client-types.ts

```diff
@@ -74,3 +74,8 @@ export interface HttpResponse extends KnownHttpResponseFields {
 export interface HttpReponse extends KnownHttpResponseFields {
   [otherKey: string]: any;
 }
+
+export interface HttpRequestAndResponse{
```

Comment:
Discussion @marikaner @jjtang1985 @florian-richter: What if we would also include a typed data object, then people can use the request and response but also have a typed data in the executeRaw response.

Reply:
I have some use cases where we can not build the type data (return entity from deserialisation), which makes this ticket in the input queue.
- function/action import with entity type as return type, that matches to multiple entity sets. We recommend to use `executeRaw`, since deserialisation is not possible.
- this [os issue](https://github.com/SAP/cloud-sdk-js/issues/682) where special response structure does not meet our expectation and therefore deserialisation is not possible.

I think typically, the best practise is to use `execute` to get an entity. But for the cases shown above or for debugging purpose, we can use `executeRaw`.

Reply:
Ok I got the point, that the `executeRaw` is also our fallback if something goes wrong with the deserialization. But I do not see the `action-function-import-request-builder`. Is it missing?

Reply:
Good catch, I'll add it.